src/unix/linux_like/linux/gnu/b64/aarch64/mod.rs

```diff
@@ -631,10 +631,6 @@ pub const PR_PAC_APGAKEY: c_ulong = 1 << 4;
 
 pub const PR_SME_SET_VL: c_int = 63;
 pub const PR_SME_GET_VL: c_int = 64;
-pub const PR_SME_VL_LEN_MAX: c_int = 0xffff;
-
-pub const PR_SME_SET_VL_INHERIT: c_ulong = 1 << 17;
-pub const PR_SME_SET_VL_ONE_EXEC: c_ulong = 1 << 18;
```

Comment:
What is the reason for removing these? I can't actually find them defined anywhere, @devnexen do you remember what these are from in https://github.com/rust-lang/libc/pull/3116?

Reply:
If it is something that makes sense to remove, I'd rather only deprecate it here so I can backport the PR. We'll go through and do a sweep of deprecated items before 1.0.